Migration to AndroidX and replaced gradle deprectated API on bintray.gradle

amilcar-sr opened the pull request.

Commit:
Migration to AndroidX and replaced gradle deprectated API on bintray.gradle

Commit:
Replaced missed imports to support library

amilcar-sr commented:
@hamidness several of the devs using your library need this PR so they can migrate to AndroidX

dbotelho commented:
@amilcar-sr I've been using your https://github.com/amilcar-sr/restring/tree/androidx-migration fork from this project, but now it doesn't work if you change the "targetSdkVersion" to "29" on Android 10 devices. Do you intend to submit a fix for this in the near future?

Commit:
Fix Q Crash

This fixed came form https://github.com/takahirom/DownloadableCalligraphy/pull/4

amilcar-sr committed:
Merge pull request #1 from dbotelho/bugfix/fix-q-crash

Fix Q Crash
